packages/hash/frontend/src/blocks/page/createEditorView.ts

```diff
@@ -61,7 +61,7 @@ export const createEditorView = (
       );
     },
     nodeViews: {
-      block(currentNode, currentView, getPos) {
+      block(currentNode, currentView, getPos, _decorations) {
```

Comment:
Might be worth adding a comment pointing to https://github.com/DefinitelyTyped/DefinitelyTyped/pull/57384#issuecomment-1018936089

Reply:
Yeah that’d be a useful reminder! I added a comment in each of the three cases.